cms/utils/compat/__init__.py

```diff
@@ -14,4 +14,4 @@
 DJANGO_3 = Version(DJANGO_VERSION) < Version('4.0')
 DJANGO_4_1 = Version(DJANGO_VERSION) < Version('4.2')
 DJANGO_4_2 = Version(DJANGO_VERSION) < Version('4.3')
-DJANGO_5_1 = Version(DJANGO_VERSION) < Version('5.2')
+DJANGO_5_1 = Version(DJANGO_VERSION) < Version('5.2.dev')  # To allow testing against django's main branch
```

Comment:
Could the version ever be 5.2rc1 or rc2? It not, then we are good. Just wondering.